hyperjumptech/monika
# Added:  prettify email notification with mailgen

# Monika Pull Request (PR)  
This PR resolves #454 

## What feature/issue does this PR add  
1. This PR is adding improvement on layout email notification

## How did you implement / how did you fix it  
1.  Using npm package named [mailgen](https://github.com/eladnava/mailgen)

## How to test  
1. set config in monika.yml with type smtp. In this case, I'm using mailnest as smtp server
2. run `npm run start`
3. see the email notification on mailnest inbox
